### PR TITLE
Update go.mod - CVE-2020-29652 - golang.org/x/crypto

### PR DESCRIPTION
**Pull request by Momotoculteur**
Hello team !

I got a CVE issue raised by Trivy in my Gitlab pipeline caused by one of your dependency :  

golang.org/x/crypto | CVE-2020-29652   | HIGH     
Used version : v0.0.0-20200302210943-78000ba7a073 
Fixed version : v0.0.0-20201216223049-8b5274cf687f

Is it a correct way to fix it with my pull request ?

Have a good day,
Bastien

### CODE REVIEW

**Review by matthewdale — COMMENTED** [review]
Hey @Momotoculteur, thanks for the PR! Just one suggestion to fix the CI build.

File: go.mod [hunk 1]
```diff
@@ -21,7 +21,7 @@ require (
 	github.com/xdg-go/scram v1.0.2
 	github.com/xdg-go/stringprep v1.0.2
 	github.com/youmark/pkcs8 v0.0.0-20181117223130-1be2e3e5546d
-	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
+	golang.org/x/crypto v0.0.0-20201216223049-8b5274cf687f
```
[the remark this answers is not in the document]
Comment (matthewdale) [reply go.mod:24]: The `go.sum` file looks good now, thanks!

File: go.mod [hunk 2]
```diff
@@ -21,7 +21,7 @@ require (
...
 	golang.org/x/sync v0.0.0-20190911185100-cd5d95a43a6e
 	golang.org/x/sys v0.0.0-20190531175056-4c3a928424d2 // indirect
```
Comment (matthewdale) [inline go.mod:26]: The `golang.org/x/sys` package in `go.mod` is out of sync with the one in `go.sum` and is no longer an indirect dependency. Please remove it from `go.mod`.

```suggestion
```

**Review by matthewdale — COMMENTED** [review]
One more suggestion to fix a new build failure.

File: go.mod (hunk 2 above)
Comment (Momotoculteur) [reply go.mod]: Should be ok now :)

**Review by matthewdale — APPROVED** [review]
Note that we still need to update the vendored dependencies to get rid of the vendored dependency version mismatch warnings, but I will do that in a subsequent PR. Otherwise, looks good 👍

**Pull request merged**